Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM node:10.22.1
+
+WORKDIR /root/app
+
+ENV TZ=Australia/Melbourne
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+RUN apt-get update \
+  && apt-get install -y jq \
+  rm -rf /var/lib/apt/lists/*
+
+RUN npm i -g npm@latest
+
+COPY package.json package-lock.json ./
+
+RUN npm ci --quiet --no-optional && \
+  rm -f ~/.npmrc && \
```

Comment:
```suggestion
```

Not needed as we don't have the npmrc file because we're not installing anything private